core/state_transition.go

```diff
@@ -44,8 +43,10 @@ The state transitioning model does all the necessary work to work out a valid ne
 3) Create a new state object if the recipient is \0*32
 4) Value transfer
 == If contract creation ==
-  4a) Attempt to run transaction data
-  4b) If valid, use result as code for the new state object
+
+	4a) Attempt to run transaction data
+	4b) If valid, use result as code for the new state object
+
```

Comment:
Should we undo these format changes? (Also a couple of other places below.)

Reply:
fixed in https://github.com/scroll-tech/go-ethereum/pull/351/commits/c7a1066892649ebecfd3a47082b0e21fc1aa85bb